traffic_ops/traffic_ops_golang/server/servers_server_capability.go

```diff
@@ -165,3 +186,15 @@ server) VALUES (
 :server_capability,
 :server) RETURNING server, server_capability, last_updated`
 }
+
+func scCheckServerTypeQuery() string {
+	return `
+SELECT EXISTS (
+	SELECT s.id
+	FROM server s
+	JOIN type t ON s.type = t.id
+	WHERE s.id = $1
+	AND t.use_in_table = 'server'
+	AND (t.name LIKE '%MID%' OR t.name LIKE '%EDGE%')
```

Comment:
I wonder if it should be:

`AND (t.name LIKE 'MID%' OR t.name LIKE 'EDGE%')`

If you look around the code base, I'm pretty sure we only consider EDGE* as an edge and not FOO_EDGE for example...

Reply:
we must not extend our system hacks! ;)

Reply:
will change :) good catch

Reply:
I agree with @mitchell852, AFAIK most or all existing things check for the prefix. 

Checking anywhere would make this strictly more lenient than most other checks, which sounds nice, but if someone relies on it, then stuff elsewhere might not work that they don't expect.

Better to require a prefix everywhere I think.

Reply:
Erm, sorry, didn't refresh to see the new comments before I posted that :|